reactor-core/src/main/java/reactor/core/publisher/EmitterProcessor.java

```diff
@@ -51,7 +52,14 @@
  * @param <T> the input and output value type
  *
  * @author Stephane Maldini
- * @deprecated Prefer clear cut usage of {@link Sinks}, to be removed in 3.5
+ * @deprecated To be removed in 3.5. Prefer clear cut usage of {@link Sinks} through
+ * variations of <pre><code>Sinks.many().multicast().onBackpressureBuffer()</code></pre>.
+ *
+ * This processor is blocking on {@link EmitterProcessor#emitNext(T)}. This behaviour can be implemented
+ * with the {@link Sinks} API by calling {@link Sinks#many()#tryEmitNext(T)} and retrying, e.g.:
+ *  <pre><code>
```

Comment:
this one I'd keep as a `<pre><code>` indeed, but maybe add newlines and indentation for readability?

Reply:
working on it. using `<pre>{@code ...}</pre>` seems to be the best solution here (see https://reflectoring.io/howto-format-code-snippets-in-javadoc/)